boards/shared/Src/Io/Io_SharedFrequencyOnlyPwmInput.c

```diff
@@ -0,0 +1,127 @@
+#include <main.h>
+#include <math.h>
+#include <assert.h>
+#include "Io_SharedFrequencyOnlyPwmInput.h"
+
+#define MAX_TIMER_PERIOD_ELAPSED_COUNT 2
+#define AUTO_RELOAD_REG 65535
```

Comment:
For `AUTO_RELOAD_REG` I would've used a cube generated constant but I didn't want to modify the `.ioc` file for each board to include a timer prescaler constant. I've stuck with this for now.

Reply:
This should just be turned into an argument that gets passed into `_Create` then

Reply:
fixed